also link with -ldstc++fs for libCommon.a, required by older versions of GCC (< 9.0)
fixes errors like:
    error: undefined reference to 'std::filesystem::space(std::filesystem::__cxx11::path const&)'
author: Kenneth Hoste (HPC-UGent)
--- openMVS-2.0.1/libs/Common/CMakeLists.txt.orig	2022-10-28 09:17:15.474116317 +0200
+++ openMVS-2.0.1/libs/Common/CMakeLists.txt	2022-10-28 09:17:22.004999058 +0200
@@ -12,7 +12,7 @@
 endif()
 
 # Link its dependencies
-TARGET_LINK_LIBRARIES(Common ${Boost_LIBRARIES} ${OpenCV_LIBS})
+TARGET_LINK_LIBRARIES(Common ${Boost_LIBRARIES} ${OpenCV_LIBS} "stdc++fs")
 
 # Install
 SET_TARGET_PROPERTIES(Common PROPERTIES
